cfg/gke-1.6.0/controlplane.yaml

```diff
@@ -0,0 +1,35 @@
+---
+controls:
+version: "gke-1.6.0"
+id: 2
+text: "Control Plane Configuration"
+type: "controlplane"
+groups:
+  - id: 2.1
+    text: "Authentication and Authorization"
+    checks:
+      - id: 2.1.1
+        text: "Client certificate authentication should not be used for users (Automated)"
+        audit: "kubectl get secrets --namespace kube-system -o json"
+        remediation: |
+          Alternative mechanisms provided by Kubernetes such as the use of OIDC should be
+          implemented in place of client certificates.
+          You can remediate the availability of client certificates in your GKE cluster. See
+          Recommendation 5.8.1.
+        scored: false
+
+  - id: 2.2
```

Comment:
There is no more section 2.2. in GKE 1.6.0: https://workbench.cisecurity.org/benchmarks/16093/sections/2382179